xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1461,6 +1461,20 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
     stream->flags = (CDemuxStream::EFlags)pStream->disposition;
 
     AVDictionaryEntry *langTag = av_dict_get(pStream->metadata, "language", NULL, 0);
+    if (!langTag)
+    {
+      if ((strcmp(m_pFormatContext->iformat->name, "avi") == 0) && (streamIdx > 0 && streamIdx < 10))
```

Comment:
After some research I found out, that this tags were introduced by Morgan Multimedia Stream Switcher, as shown in [this table](https://matroska.org/technical/specs/tagging/othertagsystems/comparetable.html). Quotation from the [website of Stream Switcher](http://www.morgan-multimedia.com/morgan/php/products.php?sProductId=10): 

> IAS1 to IAS9 indentify audio streams 1 to 9.

So you are right. It starts with stream 0, where all non-audio streams are ignored. This means only audio streams are counted (e.g. stream 1 is the first audio stream -> audio stream 0). I've pushed an updated version.